### DISCUSSION TITLE
Handlebarjs can not import data outside #each loop ?

### DISCUSSION THREAD

**Discussion by THKWjZ** [handlebars-lang/handlebars.js#1924] [Q&A]
can anybody help me solution for this case please
seem like i can not get data from database outside the each loop
im new in backend dev so please help me, thank you so much
![image](https://user-images.githubusercontent.com/126941013/223632837-3b6f538f-ed6b-45dd-89af-960ab25a061a.png)
![image](https://user-images.githubusercontent.com/126941013/223633132-aec054f2-9064-4779-b689-2902cf6d95e2.png)

**Comment by jaylinski** [comment]
I'm not sure what you are trying to do. What exactly is your problem?

You should be able to fetch data and just pass it to your handlebars template.

**Comment by satyajitnayk** [comment]
@the890vn you should be able to use the data just by passing the products to each helper.

If you could clarify what exactly you want to achieve, it would be easier to answer your query.
